Replication_v1.R

```diff
@@ -354,6 +356,204 @@ svy_mean_lcu_table_tar <- db_create_lcu_table(dl = svy_mean_lcu_tar,
                                               pfw_table = dl_aux$pfw)
 
 
+# Objective:  Local Currency Unit survey mean table 
```

Comment:
This should not be here (these are the previous functions). I will make a commit on your branch to eliminate it and test if works without it.